components/formats-gpl/matlab/bfCheckJavaPath.m

```diff
@@ -50,11 +50,12 @@
 % Check if a Bio-Formats JAR file is in the Java class path
 % Can be in either static or dynamic Java class path
 jPath = javaclasspath('-all');
-bfJarFiles = {'bioformats_package.jar', 'loci_tools.jar'};
+jars = cell2mat (cell2mat (regexp (jPath, '[\\/]{1}([^/\\]+)\.jar$', 'tokens')));
```

Comment:
This flattens a cell array of cell arrays. It works in Octave, I assumed that Matlab also supported it:

    octave:1> bfJarFiles = {'bioformats_package.jar', 'loci_tools.jar'};
    octave:2> jPath = javaclasspath('-all');
    octave:3> cell2mat (cell2mat (regexp (jPath, '[\\/]{1}([^/\\]+)\.jar$', 'tokens')))
    ans = 
    {
      [1,1] = ij
      [1,2] = bioformats_package
    }

Would you please check if the alternative syntax is supported in Matlab?

    octave:8> names = regexp (jPath, '[\\/]{1}([^/\\]+)\.jar$', 'tokens');
    octave:9> [[names{:}]{:}]
    ans = 
    {
      [1,1] = ij
      [1,2] = bioformats_package
    }